src/MainWindow.cpp

```diff
@@ -430,38 +430,42 @@ MainWindow::ReinitialiseLayout_flarm(PixelRect rc,
 
   case TrafficSettings::GaugeLocation::TOP_LEFT_AVOID_IB:
     rc.top = GetMainRect().top;
+	rc.left = GetMainRect().left;
```

Comment:
Please remove indent.

Reply:
Hope I succeeded now. With "edit in place", every try to indent gave a wrong indentation. With GitHub Desktop, the file edited locally in notepad++ looked fine. Now with I try it with GitHub Dev and the web editor.